cmd/version.go

```diff
@@ -62,3 +62,13 @@ func versionString() string {
 		goarch,
 	})
 }
+
+// CliVersionString returns only the CLI version string
+func CliVersionString() string {
```

Comment:
I think in this case, the func is specific for Kubebuilder right?
So, could we make clear it in the name 
```suggestion
func GetKubebuilderVersion() string {
```